### PR TITLE
feat(TE-125) - Inline Related Article placement

### PR DESCRIPTION
**Pull request by vinhlam**

### CODE REVIEW

File: packages/article-skeleton/src/article-body/article-body.web.js [hunk 1]
```diff
@@ -310,6 +311,19 @@ const renderers = ({
       />
     );
   },
+  autoInlineRelatedArticles(key, { element }) {
```
Comment (bibicollins1) [inline packages/article-skeleton/src/article-body/article-body.web.js:314]: Just a quick thought, are we calling these InlineRelatedArticles or InArticleRelatedArticles? Just wondering if we want to make it consistent?

File: packages/article-skeleton/__tests__/web/inline-related-article.test.js [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    });
+
+    it("outside paywall", () => {
+      const fourParagraphContenet = [
+        fakeContent(),
+        fakeContent(),
+        fakeContent(),
+        fakeContent()
+      ];
+      expect(contentReducer(fourParagraphContenet)).toEqual(
```
Comment (bibicollins1) [inline packages/article-skeleton/__tests__/web/inline-related-article.test.js:163]: Tiny thing, but think there is a misspelling in fourParagraphContenet -- fourParagraphContent?

File: packages/article-skeleton/src/contentModifiers/inline-related-article.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  },
+  children: []
+});
+
+const insertInlineRelatedArticles = (
+  relatedArticleSlice,
+  inlineRelatedArticlesFlag,
+  { afterParagraph = 7, paragraphPadding = 2 } = {}
+) => articleContent => {
+  console.log(
```
Comment (bibicollins1) [inline packages/article-skeleton/src/contentModifiers/inline-related-article.js:19]: Do we need this log?

Comment (bibicollins1) [reply packages/article-skeleton/src/contentModifiers/inline-related-article.js:19]: And the ones below.. is it for more visibility in prod articles while testing this?

**Review by bibicollins1 — APPROVED** [review]
Couple of comments about the logs, but otherwise works great! Good to see lots of article examples and test how the placement works in them 👍

File: packages/article-skeleton/src/contentModifiers/inline-related-article.js (hunk 3 above)
Comment (vinhlam) [reply packages/article-skeleton/src/contentModifiers/inline-related-article.js:19]: I think I accidentally added it while debugging some build issues. Will remove, don't think the build will let it through

**Pull request merged**